### PR TITLE
Fix metrics bug and add metric for copy chan size

### PR DESCRIPTION
**Pull request by maggie-lou**
* The same bug I fixed [here](https://github.com/buildbuddy-io/buildbuddy/pull/2789) for logging was also making the metrics look worse than is true

### CODE REVIEW

**Comment by maggie-lou** [comment]
Current dependencies on/for this PR:
* master
  * **PR #2811** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/2811" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/2811?utm_source=stack-comment).

File: enterprise/server/backends/migration_cache/migration_cache.go [hunk 1]
```diff
@@ -834,20 +836,26 @@ func (mc *MigrationCache) copyDataInBackground() error {
...
 }
 
-func (mc *MigrationCache) logCopyChanFullInBackground() {
-	ticker := time.NewTicker(mc.copyChanFullWarningInterval)
-	defer ticker.Stop()
+func (mc *MigrationCache) logCopyChanInfoInBackground() {
+	copyChanFullTicker := time.NewTicker(mc.copyChanFullWarningInterval)
+	defer copyChanFullTicker.Stop()
+
+	metricTicker := time.NewTicker(5 * time.Minute)
```
Comment (vadimberezniker) [inline enterprise/server/backends/migration_cache/migration_cache.go:843]: There's no harm in making this much finer, say every 5 seconds. 5 minutes might be enough time to burst and drain a lot of copies from the queue which would not really give you a good insight into the behavior of the queue.

File: enterprise/server/backends/migration_cache/migration_cache.go [hunk 2]
```diff
@@ -834,20 +836,26 @@ func (mc *MigrationCache) copyDataInBackground() error {
 	}
 }
 
-func (mc *MigrationCache) logCopyChanFullInBackground() {
-	ticker := time.NewTicker(mc.copyChanFullWarningInterval)
-	defer ticker.Stop()
+func (mc *MigrationCache) logCopyChanInfoInBackground() {
```
Comment (vadimberezniker) [inline enterprise/server/backends/migration_cache/migration_cache.go:839]: what do you think about calling this `monitorCopyChanFullness`?

**Review by vadimberezniker — APPROVED** [review]

File: enterprise/server/backends/migration_cache/migration_cache.go (hunk 2 above)
Comment (maggie-lou) [reply enterprise/server/backends/migration_cache/migration_cache.go]: Like that - fixed

File: enterprise/server/backends/migration_cache/migration_cache.go (hunk 1 above)
Comment (maggie-lou) [reply enterprise/server/backends/migration_cache/migration_cache.go]: Ack

**Pull request merged**